core/templates/components/forms/forms-templates/mark-translations-as-needing-update-modal.component.ts

```diff
@@ -30,15 +33,34 @@ export class MarkTranslationsAsNeedingUpdateModalComponent extends ConfirmOrCanc
   @Input() markNeedsUpdateHandler!: (contentId: string) => void;
   @Input() removeHandler!: (contentId: string) => void;
 
-  constructor(private ngbActiveModal: NgbActiveModal) {
+  modifyTranslationsFeatureFlagIsEnabled: boolean = false;
+
+  constructor(
+    private ngbActiveModal: NgbActiveModal,
+    private ngbModal: NgbModal,
+    private platformFeatureService: PlatformFeatureService
+  ) {
     super(ngbActiveModal);
   }
 
+  ngOnInit(): void {
+    this.modifyTranslationsFeatureFlagIsEnabled =
+      this.platformFeatureService.status.ExplorationEditorCanModifyTranslations.isEnabled;
```

Comment:
Should we wrap in `(..)` per https://github.com/oppia/oppia/wiki/Coding-style-guide#javascript?

Reply:
I don't believe its needed here and it is pretty standard to have it this way in the codebase, would be needed if the line to the right side of `=` was further split up into multiple lines

Reply:
Feel free to ignore wrapping here in case adding parenthesis ends up causing L48 to hit the line character limit.

Reply:
@kevintab95 I tried adding the brackets but they got removed automatically by Prettify itself, I assume it is the standard for such lines...?

Reply:
I tried a few variations on my machine. From what I can tell, the pre-commit linter does not care even if the line is too long:
![Screenshot from 2024-06-16 11-42-48](https://github.com/oppia/oppia/assets/11008603/5d35bc67-0b23-4e18-ba38-1b0458ed3cd7)


I was able to successfully commit these lines and no modifications were made by prettier.

Reply:
Interesting, I tried this variation which seems to revert back 
<img width="713" alt="image" src="https://github.com/oppia/oppia/assets/73329256/d648ce49-9a87-499a-aa50-b7bbad3c4ed3">
<img width="544" alt="image" src="https://github.com/oppia/oppia/assets/73329256/92d72477-0b16-45fa-8614-6ca9fb8f4b0f">

Reply:
@nikitaevg do you know why this is happening? i.e. prettier seems to work differently on our machines.

Reply:
Hmm, no idea TBH. You sure you did save the file? The commit stats say you added 1 line and modified 1 line, but the screenshot suggests much more changes